code/ATMOSPHERICS/pipes.dm

```diff
@@ -604,12 +600,8 @@
 	color = PIPE_COLOR_GREEN
 /obj/machinery/atmospherics/pipe/manifold/insulated
 	name = "\improper Insulated pipe"
-	//icon = 'icons/obj/atmospherics/red_pipe.dmi'
-	icon_state = "manifold"
```

Comment:
The rest of the PR was reducing var overrides that were already the same on the parent.